examples/tv-casting-app/tv-casting-common/core/CommissionerDeclarationHandler.cpp

```diff
@@ -36,14 +38,36 @@ CommissionerDeclarationHandler * CommissionerDeclarationHandler::GetInstance()
 }
 
 // TODO: In the following PRs. Implement setHandler() for CommissionerDeclaration messages and expose messages to higher layers for
-// Linux, Android and iOS.
+// Linux(DONE), Android(pending) and iOS(pending).
```

Comment:
For future reference, task tracking commentary should just be added in the PR description.